packages/calcite-components/src/components/action/resources.ts

```diff
@@ -1,12 +1,16 @@
 export const CSS = {
+  actionIndicator: "action-indicator",
   button: "button",
-  buttonTextVisible: "button--text-visible",
   buttonCompact: "button--compact",
-  indicatorText: "indicator-text",
+  buttonTextVisible: "button--text-visible",
   iconContainer: "icon-container",
+  indicatorText: "indicator-text",
+  indicatorWithIcon: "indicator-with-icon",
```

Comment:
Nitpick: `--` should be used as a delimiter to state/modifiers. Also, one could be dropped if either with or without icon is treated as the default (e.g., `indicator` + `indicator--icon`).

Reply:
These are the old classes. Would you like me to add these as a refactor within this PR?

Reply:
Gotcha. We can do this separately then. Might be good to do a quick audit just to see how we're doing there. Very low priority though.